gazelle/js/parser/parser.go

```diff
@@ -53,7 +53,7 @@ var importQueries = map[string]string{
 	`,
 }
 
-var tripleSlashRe = regexp.MustCompile(`^///\s*<reference\s+(?:lib|path|types)\s*=\s*"(?P<lib>[^"]+)"`)
+var tripleSlashRe = regexp.MustCompile(`^///\s*<reference\s+(?P<attr>lib|path|types)\s*=\s*"(?P<lib>[^"]+)"`)
```

Comment:
Can we just remove `lib|` from here?

Reply:
I think so. I wasn't sure whether you'd prefer to parse it and then explicitly ignore it or just pretend like we never saw it. Defer to you!

Reply:
Updated!